scripts/cs10/caching.js

```diff
@@ -315,6 +319,7 @@ function sendAsFileOrMsg(text, fileName, msg) {
     // Files can only be sent when using the hipchat adapter
     var filePath = './temp1234';
     if (robot.adapterName == 'hipchat') {
+        // TODO: Replace this with Buffers
```

Comment:
I think I put that in because Buffers are more memory efficient, but since the command is async, and the files are small, we probably don't need to worry.